common/src/main/java/uk/gov/gchq/palisade/service/request/Request.java

```diff
@@ -30,27 +30,45 @@
  * This makes sure each request has a unique identifier.
  */
 public abstract class Request {
-    private String id;
+    private String id; //this is a unique ID for each individual request made between services
+    private String originalRequestId; //this Id is unique between requests
```

Comment:
I would extend the description for originalRequestId to make it clear that it is unique per data access request from a user while the id field is unique for every request between the various micro-services